src/Data/Foreign.purs

```diff
@@ -81,6 +81,8 @@ renderForeignError (TypeMismatch exp act) = "Type mismatch: expected " <> exp <>
 
 -- | An error monad, used in this library to encode possible failures when
 -- | dealing with foreign data.
+-- | `Except e` was chosen instead of `Either e` because `Except`'s `Alt` instance
```

Comment:
Also, nitpick: instead of "was chosen", I would prefer to phrase this more like "The `Alt` instance for `Except` allows us to accumulate multiple errors, unlike `Either`, or something along those lines.